bower.json

```diff
@@ -68,6 +68,7 @@
     "summernote-0-8-2": "summernote#0.8.2",
     "underscore-1-8-3": "underscore#1.8.3",
     "undone.js-0-0-1": "https://github.com/yckart/undone.js.git#0.0.1",
-    "validate.js-0-10-0": "validate.js#0.10.0"
+    "validate.js-0-10-0": "validate.js#0.10.0",
+    "bootstrap-sidebar": "^0.2.2"
```

Comment:
I learned early on the bower project that we need to maintain strict control of the versions to make it work (bower is simply not made for complicated legacy projects), so should follow the formatting as above(display this limitation, bower is still awesome for standardizing/tracking/centralizing assets). See here for the bower wiki page where track the assets:
http://www.open-emr.org/wiki/index.php/Bower